### ISSUE TITLE
Missing AddArrowFunctionReturnTypeRector

### ISSUE THREAD

**Issue by MatusBoa** [rectorphp/rector#7502]
# Bug Report

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | 0.14.2            |

According to docs, there should be `AddArrowFunctionReturnTypeRector` but for some reason I cannot find it. (https://github.com/rectorphp/rector/blob/main/docs/rector_rules_overview.md#addarrowfunctionreturntyperector)

Am I missing something?

<img width="874" alt="image" src="https://user-images.githubusercontent.com/8343385/191591489-74d6d080-9387-444e-a6b3-e425280cc89b.png">

**Comment by TomasVotruba** [comment]
Hi, 

what version was the `Rector\TypeDeclaration\Rector\ArrowFunction\AddArrowFunctionReturnTypeRector` added in?

**Comment by samsonasik** [comment]
AddArrowFunctionReturnTypeRector is on dev-main, you can try:

```
composer config minimum-stability dev
composer config prefer-stable true
composer require --dev rector/rector:dev-main

**Comment by samsonasik** [comment]
Released 0.14.3 for it https://github.com/rectorphp/rector/releases/tag/0.14.3
